packages/angular_devkit/build_angular/src/webpack/configs/styles.ts

```diff
@@ -299,20 +301,39 @@ export function getStylesConfig(wco: WebpackConfigOptions): webpack.Configuratio
     );
 
     extraMinimizers.push(
+      // Component styles use esbuild which is faster and generates smaller files on average.
+      // esbuild does not yet support style sourcemaps but component style sourcemaps are not
```

Comment:
Can we add a comment to reference https://github.com/evanw/esbuild/issues/519 to traceability?